src/instance.rs

```diff
@@ -1516,9 +1514,8 @@ impl<T> Py<T> {
     ) -> PyResult<PyObject>
     where
         N: IntoPyObject<'py, Target = PyString>,
-        A: IntoPyObject<'py, Target = PyTuple>,
+        A: IntoPy<Py<PyTuple>>,
```

Comment:
@Icxolu I reverted the bounds here and on the other call functions given the likely plan is to more these bounds to a separate trait.